cypress/integration/purchase/purchase_order_complete_change_complete.js

```diff
@@ -0,0 +1,170 @@
+        .setName(discountSchemaName)
+        .apply();
+    });
+    Builder.createBasicProductEntitiesWithPrice(priceListName, productForPackingMaterial, productPMValue, productType);
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialName)
+        .setProduct(productForPackingMaterial)
+        .apply();
+    });
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsName)
+        .apply();
+    });
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsName)
+        .setPackingInstructions(packingInstructionsName)
+        .setPackingMaterial(packingMaterialName)
+        .apply();
+    });
+    cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
+      Object.assign(new ProductCategory(), productCategoryJson)
+        .setName(productCategoryName)
+        .setValue(productCategoryValue)
+        .apply();
+    });
+
+    Builder.createBasicProductEntitiesWithCUTUAllocation(
+      productCategoryName,
+      productCategoryValue,
+      priceListName,
+      productName1,
+      productValue1,
+      productType,
+      packingInstructionsName
+    );
+    cy.fixture('sales/simple_vendor.json').then(vendorJson => {
+      new BPartner({ name: vendorName })
+        .setVendor(true)
+        .setVendorPricingSystem(priceSystemName)
+        .setVendorDiscountSchema(discountSchemaName)
+        .setPaymentTerm('30 days net')
+        .addLocation(new BPartnerLocation('Address1').setCity('Cologne').setCountry('Deutschland'))
+        .apply();
+    });
+    cy.readAllNotifications();
+  });
+  it('Create a purchase order and complete it', function() {
+    cy.visitWindow('181', 'NEW');
+    cy.get('#lookup_C_BPartner_ID input')
+      .type(vendorName)
+      .type('\n');
+    cy.contains('.input-dropdown-list-option', vendorName).click();
+
+    cy.selectInListField('M_PricingSystem_ID', priceSystemName, false, null, true);
+    const addNewText = Cypress.messages.window.batchEntry.caption;
+    cy.get('.tabs-wrapper .form-flex-align .btn')
+      .contains(addNewText)
+      .should('exist')
+      .click();
+    cy.get('.quick-input-container .form-group').should('exist');
+    cy.writeIntoLookupListField('M_Product_ID', productName1, productName1, false, false, null, true);
+
+    cy.get('.form-field-Qty')
+      .click()
+      .find('.input-body-container.focused')
+      .should('exist')
+      .find('i')
+      .eq(0)
+      .click();
+    cy.server();
+    cy.route('POST', `/rest/api/window/${purchaseOrders.windowId}/*/${purchaseOrders.orderLineTabId}/quickInput`).as(
+      'resetQuickInputFields'
```

Comment:
looks like this alias isn't used; do we need it?

Reply:
yes, I have tried to use it in a wait: e.g: cy.wait('@resetQuickInputFields'); not to use wait(5000) but it didn't work as I got: CypressError: Timed out retrying: cy.wait() timed out waiting 30000ms for the 1st request to the route: 'resetQuickInputFields'. No request ever occurred.

Reply:
ok, then pls either check out  the frontend does and why the route is not matched fails (e.g. maybe it's not a post, or maybe the URL doesn't match) and fix it, 
..or remove it

Reply:
ok, will do

Reply:
removed it in all the tests in which was used.